vocdoni/bridge-ui
# Fix/proposal creation dao 109

# Short description
- Fixes icon loading error
- Fixes button layout on certain screen sizes

# How to test
- Create a proposal on any Token. 
- Reject signing the transaction.
- On the creation dialog, make sure that there is a red cross icon
- Make sure that the two buttons have a proper layout 

# Tracker ID
DAO-109
